src/runtime/refiner.ts

```diff
@@ -476,7 +496,27 @@ export class Range {
         newRange.segments.push(Segment.overlap(seg, subRange));
       }
     }
-    this._segments = newRange.segments;
+    this.segments = newRange.segments;
+  }
+
+  // This function assumes that the expression is univariate
+  // and has been normalised (see above for definition).
```

Comment:
> (see above for definition)

It's not really clear what this is referring to... there are no comments above talking about the normalised form, and it's not obvious what method to look at.

Reply:
I was just referring to the normalise function under Refinement. It has a some comments explaining how that function normalises an expression. Do you think I should be more explicit by saying 'see Refinement.normalise()' for definition?

Reply:
That would be better, yes.